### ISSUE TITLE
PA

### ISSUE THREAD

**Issue by turicas** [turicas/covid19-br#24]
Boletins:

- [Link para o site dos boletins na Secretaria de Saúde de PA](http://www.saude.pa.gov.br/category/coronavirus/)

Status da checagem (anteriores a 20/03):

- [ ] Boletins
- [ ] Casos

Modo e responsável pela conversão:

- Boletins: manualmente por @turicas
- Casos: manualmente por ?

**Comment by dhonysilva** [comment]
Ainda não entendi bem como posso inserir os casos manualmente? É na planilha mesmo? Devo duplicar a aba template, renomear para 'PA' e digitar os dados?

Ainda não tenho permissão na planilha pra fazer isso.

**Comment by turicas** [comment]
@dhonysilva por enquanto cria uma planilha sua onde vc quiser e só segue o padrão que estabeleci na planilha de casos. Estou reestruturando tudo e em breve te dou acesso à planilha do PA (me manda seu whatsapp por email - turicas arroba brasil ponto io).

**Comment by raphageoc** [comment]
A sespa não esta divulgando nos boletins os municípios dos casos e na página do facebook dele tem um comunicado de um caso em Maraba, porém os outros 4 casos não tem localização oficial.

**Comment by dhonysilva** [comment]
@raphageoc isso. Por isso estou anotando todos os casos como "Total do Estado", com exceção do caso de Marabá.

**Comment by raphageoc** [comment]
@dhonysilva achei a fonte oficial de dois casos de belem 
https://www.agenciapara.com.br/noticia/18561

**Comment by dhonysilva** [comment]
Obrigado.

Espalhado em notícias e não oficialmente no Boletim. Vou juntar essas evidências pra fazer uma reclamação formal.

**Comment by raphageoc** [comment]
@dhonysilva  Sim, embora seja um meio oficial do governo não há uma centralização das informações, achei mais um caso de belem: https://www.agenciapara.com.br/noticia/18513/.
Estou procurando nas noticiias da secretaria de comunicação e vou colocando aqui.

**Comment by raphageoc** [comment]
A informação do primeiro caso, em belém tambem:
https://www.agenciapara.com.br/noticia/18484/

**Comment by raphageoc** [comment]
Os dois novos casos reportados pela sespa no facebook: https://www.facebook.com/photo/?fbid=2816848691728433&set=a.618726104874047&__cft__[0]=AZXk_JxQtgpwK--4iqL0D4KF42ns84UPM-wGY89oj_AO1eFmgY9nxAuQt-hk4V2dKbLcink0-8lQ5e4ElJpZ0q7EPZTvFFusoMwCrhO_Rv_DbVHNs8fNWROfaF5Mx-bGNbWl8LDQHa3JqpF38it1o3Vzmf8ntKM5jWLSpI_nr8OM8g&__tn__=EH-R
